python/pyspark/sql/connect/client/logging.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
Seems like linter complains that there's no license header :-).

Reply:
Thanks :). Forgot about that. Done.